docs/hugo/content/guide/authentication/credential-format.md

```diff
@@ -779,5 +779,81 @@ spec:
 EOF
 ```
 
+## User Assigned Identity Credentials
+
+### General
+This option is only available for 1st party Microsoft applications who have access to the msi data-plane.
```

Comment:
minor: maybe bold this

Reply:
Actually even better, can you put it into this:

```suggestion
{{% alert title="Warning" color="warning" %}}
This option is only available for 1st party Microsoft applications who have access to the msi data-plane.
{{% /alert %}}
```

That will make it really obvious.